src/custom-element.js

```diff
@@ -767,8 +767,16 @@ function createBaseAmpElementProto(win) {
     if (this.isInTemplate_) {
       return;
     }
-    this.resources_.remove(this);
-    this.implementation_.detachedCallback();
+    try {
+      this.implementation_.viewportCallback(false);
+    } finally {
+      try {
+        this.implementation_.unlayoutCallback();
```

Comment:
In general, I'm not sure if that's done automatically by `Resources` or if owner should do that in its own `#unlayoutCallback`. But we're not calling it anyways.